Adjust the folders of change logs

janhommes opened the pull request:
Up until now, the change logs have been created in the root folder. I now moved the files into the correct folder and the automation should now open them in the correct one.

Additionally I added the dates for all app enablement change logs, as they are all released in zone-1 (eu-latest). Is this correct this way @BeateRixen or should I wait for the automation?

BeateRixen commented:
> Up until now, the change logs have been created in the root folder. I now moved the files into the correct folder and the automation should now open them in the correct one.
> 
> Additionally I added the dates for all app enablement change logs, as they are all released in zone-1 (eu-latest). Is this correct this way @BeateRixen or should I wait for the automation?

Looks good to me. Shall I merge?

janhommes commented:
> > Up until now, the change logs have been created in the root folder. I now moved the files into the correct folder and the automation should now open them in the correct one.
> > Additionally I added the dates for all app enablement change logs, as they are all released in zone-1 (eu-latest). Is this correct this way @BeateRixen or should I wait for the automation?
> 
> Looks good to me. Shall I merge?

yes, thanks.
